### PR TITLE
Create mobile menu feature pull request #9

### PR DESCRIPTION
**Pull request by efmomoh**
- [ ] In this section of the project's pull request, the mobile menu feature was added as well as JavaScript functionality to toggle the hamburger menu when the user clicked on it.
- [ ] Adds dot eslintrc.json file to the project for clean script code.

### CODE REVIEW

File: javaScripts/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+// Add an event lister to hamburger button and the menu toogle buttons
+hamburger.addEventListener('click', () => {
+  hamburger.classList.toggle('active');
+  menu.classList.toggle('active');
+});
+
+// toogle the active class to show/hide the menu 
+document.querySelectorAll('.menu').forEach((n) => n.addEventListener('click', () => {
+  hamburger.classList.remove('active');
+  menu.classList.remove('active');
```
Comment (divinecharlotte) [inline javaScripts/index.js:14]: - [ ] kindly consider making sure that if a user clicks on one of the nav-links the correct part of your website is displayed to meet the below requirement cause currently it's not displaying the right portfolio part
> When the user clicks (or taps) any of the mobile menu options, the correct part of the page is displayed.
hint: you can do so in your HTML file by adding the correct id to each section on each nav link li

File: index.html [hunk 2]
```diff
@@ -10,13 +10,17 @@
...
+                <span class="bar"></span>
+                <span class="bar"></span>
+            </div>
             <ul class="menu">
-                <li class="list"><a href="">Portfolio</a></li>
-                <li class="list"><a href="">About</a></li>
-                <li class="list"><a href="">Contact</a></li>
+                <li class="list"><a href="#">Portfolio</a></li>
+                <li class="list"><a href="#">About</a></li>
+                <li class="list"><a href="#">Contact</a></li>
```
Comment (divinecharlotte) [inline index.html]: - [ ] kindly consider giving these nav links the same font size as given in the Figma design to meet the below design requirements
> The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).The hamburger button and menu follow the design from [the chosen template in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) (e.g., font, colors, images, text, margins).

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Team   , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters are passing👍 
- good usage of the given colors in figma👍 

 




 

## Required Changes ♻️

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 3]
```diff
@@ -557,15 +539,73 @@ label {
...
+  }
+
+  .list {
+    margin: 2rem;
+    list-style-type: none;
+  }
+
+  .list a {
+    color: #fff;
+  }
```
Comment (joel-kalema) [inline style.css]: - [ ] Good job so far, however, as mentioned by the previous reviewer, consider increasing the font size to stick to the design as much as possible. It'll be something like this:

````css

.list a {
    color: #fff;
    font-size: 32px;
    font-weight: 600;
}

````

Kindly, fix it 👍

File: index.html [hunk 4]
```diff
@@ -10,13 +10,17 @@
 </head>
 <body>
     <header class="header">
```
Comment (joel-kalema) [inline index.html:12]: - [ ] [OPTIONAL] Kindly, make sure you fixed the margins, to stick the design 👍 

> | | img | | img | |
> |-|---------|-|--------|-|
> | |Screenshot 2023-07-24 170559| |Screenshot 2023-07-24 170559| |

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @efmomoh 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

0bd223a2b156e0eaf61a713e84c3d3b7



## To highlight :champagne: 

- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by IngDavidTM — APPROVED** [review]
Hi @efmomoh 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**